src/components/elements/InstructionsUninstall.jsx

```diff
@@ -0,0 +1,33 @@
+import { loremIpsum } from 'react-lorem-ipsum'
+import logoPlaceHolder from '../../images/ellipse61.png'
+
+
+export default function Instructions() {
+  return (
+    <>
```

Comment:
We don't need the react fragment here since there is only 1 parent dom element (the ol)

Reply:
Taken out of both the Instructions and InstructionsUninstall components